api/models/Group.js

```diff
@@ -10,12 +10,14 @@ import HasSettings from './mixins/HasSettings'
 import findOrCreateThread from './post/findOrCreateThread'
 import { groupFilter } from '../graphql/filters'
 import { inviteGroupToGroup } from '../graphql/mutations/group.js'
-
 import DataType, {
   getDataTypeForInstance, getDataTypeForModel, getModelForDataType
 } from './group/DataType'
 import convertGraphqlData from '../graphql/mutations/convertGraphqlData'
 import { findOrCreateLocation } from '../graphql/mutations/location'
+import { es } from '../../lib/i18n/es'
```

Comment:
this is what should ultimately be in hylo-shared no? but ok for now to have it here

Reply:
These are only server-side strings, and they don't use the same convention as evo. I don't see any value in having these strings in shared.

There might be overlap of strings between evo and mobile, and they should be synced up on using the same framework, so I can see the value in them sharing something in shared.